src/index.html

```diff
@@ -7,6 +7,239 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="head">
+      <img class="head__moyo"
+      src="images/logo.png" alt="moyo">
```

Comment:
```suggestion
      <img
        class="head__moyo"
        src="images/logo.png"
        alt="moyo"
      >
```
Важко розібратись у атрибутах, коли вони записані так хаотичо. Форматування - це важливо. Кожен атрибут (якщо їх більше трьох), має бути на новому рядку

Reply:
Змiнено